arcgis_map_sdk_android/android/src/main/kotlin/dev/fluttercommunity/arcgis_map_sdk_android/util/GraphicsParser.kt

```diff
@@ -120,14 +123,40 @@ class GraphicsParser {
         private fun parsePictureMarkerSymbol(map: Map<String, Any>): Symbol {
             val payload = map.parseToClass<PictureMarkerSymbolPayload>()
 
-            return PictureMarkerSymbol(payload.url).apply {
+            // return local asset in case its a local path
+            if(!payload.assetUri.isWebUrl()) {
```

Comment:
Please apply IntelliJ auto-formatting in kotlin files

Reply:
✅